<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PencilUI</class>
 <widget class="DrawToolsUI" name="PencilUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>576</width>
    <height>291</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PencilUI</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>画笔大小</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="sizeSlider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sizeInput"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>画笔浓度</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="alphaSlider">
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="alphaInput"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>DrawToolsUI</class>
   <extends>QWidget</extends>
   <header>drawtoolsui.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
